### DISCUSSION TITLE
Zooz S2 Stick 700 - Controller is unable to transmit

### DISCUSSION THREAD

**Discussion by renatohtpc** [zwave-js/zwave-js-ui#3911] [General]
I am running version 9.20 on a docker. My controller is a Zooz S2 Stick 700. Following a power outage, the UI is now displaying the following error message:  "Controller is unable to transmit". Any suggestions on what may have caused this?

Thank you
Renato

**Comment by kpine** [comment]
Try physically removing the USB stick and re-inserting it. Power outages tend to do weird things with the USB sticks and cause them to lockup.

**Comment by renatohtpc** [comment]
I have tried that several times. Even tried rebooting the machine as well. I "may" have found the problem (fingers crossed). One of the switches appears to have been fried as a result of the outage. Using the replace option, I replaced it with a new one and I am no longer getting that error message.  Oddly enough, I replaces it with a Zooz 800 series. During the replace procedure, it never asked me for the key. So now it shows security "-" in red.

Thank you
Renato
